tests/python/frontend/tflite/test_forward.py

```diff
@@ -650,6 +693,51 @@ def test_all_resize():
         _test_resize(tf.image.resize_nearest_neighbor, data, align_corners=False)
 
 
+#######################################################################
+# Range
+# -------------
+def test_range():
+    # tflite 1.13 convert method does not accept empty shapes
+    if package_version.parse(tf.VERSION) >= package_version.parse('1.14.0'):
+        tf.reset_default_graph()
+        with tf.Graph().as_default():
+            data = np.array([1, 18, 3], dtype=np.int32)
+            start = tf.placeholder(dtype=tf.int32, shape=(), name="start")
+            limit = tf.placeholder(dtype=tf.int32, shape=(), name="limit")
+            delta = tf.placeholder(dtype=tf.int32, shape=(), name="delta")
+            out = tf.range(start, limit, delta, tf.int32, name="range")
+            compare_tflite_with_tvm(
+                [x for x in np.nditer(data)],
+                ["start", "limit", "delta"],
+                [start,limit,delta],
+                [out],
+                mode="vm",
+                quantized=False
+        )
+#######################################################################
+# Shape
+# -------------
+def test_shape():
```

Comment:
keep the naming convention same as other cases, test_forward_shape(), same for range also.

Reply:
Fixed...